app/views/description_change_validation_requests/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<% add_parent_breadcrumb_link "Home", planning_applications_path %>
+<% add_parent_breadcrumb_link "Application", validate_form_planning_application_path(@planning_application) %>
+<% content_for :title, "Description change" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l">
+      Request approval to a description change
+    </h2>
+    <p class="govuk-body">
+      <strong>At:</strong> <%= @planning_application.full_address  %> <br>
+      <strong>Date received:</strong> <%= @planning_application.created_at.strftime("%e %B %Y") %> <br>
+      <strong>Application number:</strong> <%= @planning_application.reference  %>
+    </p>
+
+    <div>
+      <p class="govuk-body"> <strong> Previous description:</strong><br>
+        <%= @planning_application.description  %>
+      </p>
+      <p class="govuk-body"> <strong> Proposed description:</strong><br>
+        <%= @description_change_request.proposed_description  %>
+      </p>
+    </div>
+    <p class="govuk-body">
+      <% if @description_change_request.state == "open" %>
```

Comment:
anything `state` you should never update or compare directly; use the state machine for that with `.open?` and `.some_event!` to update the states (I know we still do it in the API controllers but we'll get to that at some point).